crates/ingress-http/src/handler/error.rs

```diff
@@ -51,7 +57,7 @@ pub(crate) enum HandlerError {
     Body(anyhow::Error),
     #[error("unavailable")]
     Unavailable,
-    #[error("not ready")]
+    #[error("invocation was found, but did not complete yet")]
```

Comment:
This is a touch more grammatically correct (the word "yet" implies present perfect like _has completed_, not simple past tense 🤓). I think this is simpler:

```suggestion
    #[error("the invocation exists but has not completed yet")]
```